projects/objectnav_baselines/models/object_nav_models.py

```diff
@@ -125,71 +139,51 @@ def get_object_type_encoding(
             observations[self.goal_sensor_uuid].to(torch.int64)
         )
 
-    def forward(  # type:ignore
-        self,
-        observations: ObservationType,
-        memory: Memory,
-        prev_actions: torch.Tensor,
-        masks: torch.FloatTensor,
-    ) -> Tuple[ActorCriticOutput[DistributionType], Optional[Memory]]:
-        """Processes input batched observations to produce new actor and critic
-        values. Processes input batched observations (along with prior hidden
-        states, previous actions, and masks denoting which recurrent hidden
-        states should be masked) and returns an `ActorCriticOutput` object
-        containing the model's policy (distribution over actions) and
-        evaluation of the current state (value).
-
-        # Parameters
-        observations : Batched input observations.
-        memory : `Memory` containing the hidden states from initial timepoints.
-        prev_actions : Tensor of previous actions taken.
-        masks : Masks applied to hidden states. See `RNNStateEncoder`.
-        # Returns
-        Tuple of the `ActorCriticOutput` and recurrent hidden state.
-        """
```

Comment:
Can you add a doc string like this to the forward call of `VisualNavActorCritic`?